File: 프로그래머스/lv1/12930. 이상한 문자 만들기/이상한 문자 만들기.py
def solution(s):
    answer = []
    
    s = s.split(' ')
    for word in s:        
        tmp_word = ''
        for w in range(0,len(word)):
                
            if w % 2 == 0:
                # 짝수일 경우 → 대문자
                tmp_word += word[w].upper()
            else:
                # 홀수일 경우 → 소문자
                tmp_word += word[w].lower()

        answer.append(tmp_word)
    
    answer = ' '.join(answer)
    
    
    return answer

File: 프로그래머스/lv1/12930. 이상한 문자 만들기/이상한 문자 만들기.java
import java.util.ArrayList;

class Solution {
    public String solution(String s) {
        String answer = "";
        ArrayList<String> strArrayList = new ArrayList<>();
        
        String[] strArr = s.split(" ");
        
        for(String tmp : strArr){
            System.out.print(">>>" + tmp);
        }
        
        
        for(String word : strArr){
            String tmp_word = "";
            for(int i = 0; i < word.length(); i++){
                if(i % 2 == 0){
                    tmp_word += (word.charAt(i) + "").toUpperCase();
                }else if(i % 2 != 0){
                    tmp_word += (word.charAt(i) + "").toLowerCase();
                }else if((word.charAt(i) + "").equals(" ")){
                    System.out.println("hello world~!");
                }
                
            }
            strArrayList.add(tmp_word);
            
        }
        
        
        answer = String.join(" ", strArrayList);
        
        return answer;
    }
}